components/feature/media/src/main/java/mozilla/components/feature/media/middleware/MediaMiddleware.kt

```diff
@@ -90,7 +90,8 @@ class MediaMiddleware(
             is MediaAction.RemoveMediaAction,
             is MediaAction.RemoveTabMediaAction,
             is MediaAction.UpdateMediaVolumeAction,
-            is MediaAction.UpdateMediaStateAction -> {
+            is MediaAction.UpdateMediaStateAction,
+            is MediaAction.UpdateMediaFullscreenAction -> {
```

Comment:
I wonder if it is a bit too aggressive to run the whole aggregator whenever the fullscreen mode changes (since that should not affect the media state) - and also vise versa looking up the fullscreen mode whenever we process a playing mode change.
But then again a large part of that code may change soon with the new GeckoView API anyways - and also the fullscreen state doesn't change all the time either. So probably fine. Also, I guess, not easy to isolate since the aggregator always creates a new aggregate.